app/config/index.js

```diff
@@ -16,4 +18,5 @@ nconf
   .file('envConfig', path.join(__dirname, `${env}.json`))
   .defaults(defaults);
 
-module.exports = nconf;
+nconf.set('mailer:user', process.env.EMAIL_USER);
+nconf.set('mailer:pass', process.env.EMAIL_PASS);
```

Comment:
move to config

Reply:
[Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/20/commits/f5f1e52333bd5100b40935f350b7a08a372abf79#diff-a18e55398c3a9a4054717c2ccfa889fdR16)

Reply:
I can't make email and pass public. I think it's not secure

Reply:
so, please make corresponding changes in jenkins